### PR TITLE
POWR-821 removing information content type

### PR DESCRIPTION
**Pull request by radmosley**

### CODE REVIEW

**Comment by radmosley** [comment]
Wanted to run the test suite after removing the first content type to ensure nothing was broke.

File: web/sites/default/config/key.key.solr_core.yml [hunk 1]
```diff
@@ -9,7 +9,7 @@ key_type: authentication
 key_type_settings: {  }
 key_provider: file
 key_provider_settings:
-  file_location: sites/default/files/private/solr_core.key
```
Comment (joshuami) [inline web/sites/default/config/key.key.solr_core.yml]: This should not have changed. You may need to cache clear before running a cim. You can revert these changes individually and run cim again.

File: web/sites/default/config/core.extension.yml [hunk 2]
```diff
@@ -16,6 +16,7 @@ module:
   chosen_field: 0
   chosen_lib: 0
   ckeditor: 0
+  clamav: 0
```
Comment (joshuami) [inline web/sites/default/config/core.extension.yml]: This should not have changed. Please revert.

File: web/sites/default/config/clamav.settings.yml [hunk 3]
```diff
@@ -0,0 +1,15 @@
+enabled: true
```
Comment (joshuami) [inline web/sites/default/config/clamav.settings.yml]: This file should not have changed. Please revert.

**Comment by jookyg** [comment]
I'm marking this PR closed. It got merged but caused an error in Dev in search. There was one field from the deleted content types that hadn't been removed from the search config. I made a hotfix in branch 821-1 and deployed it, and all is working as expected now.
